azext_edge/edge/vendor/clients/deviceregistrymgmt_v2/__init__.py

```diff
@@ -0,0 +1,26 @@
+# coding=utf-8
```

Comment:
The deviceregistrymgmt_v2 client should be deleted in favor of the rest client and mechanisms in dev

Reply:
I was on it